admin-ui/plugins/auth-server/components/Agama/AgamaListPage.js

```diff
@@ -24,9 +24,19 @@ import CircularProgress from "@mui/material/CircularProgress";
 import InfoIcon from "@mui/icons-material/Info";
 import AgamaProjectConfigModal from "./AgamaProjectConfigModal";
 import { updateToast } from "Redux/features/toastSlice";
-import { isEmpty } from "lodash";
+import { isEmpty, set } from "lodash";
 import { getJsonConfig } from "Plugins/auth-server/redux/features/jsonConfigSlice";
 import SettingsIcon from "@mui/icons-material/Settings";
+import Checkbox from "@mui/material/Checkbox";
+import FormGroup from "@mui/material/FormGroup";
+import FormControlLabel from "@mui/material/FormControlLabel";
+import FormLabel from "@mui/material/FormLabel";
+import GluuTabs from "Routes/Apps/Gluu/GluuTabs";
+import { toast } from "react-toastify";
+import {
+  getAgamaRepository,
+  getAgamaRepositoryFile,
+} from "../../redux/features/agamaSlice";
```

Comment:
we should use alias here for example: https://github.com/GluuFederation/flex/blob/main/admin-ui/plugins/auth-server/components/Agama/AgamaListPage.js#L28C32-L28C66